src/styles/images/index.md.njk

```diff
@@ -8,53 +8,144 @@ layout: layout-pane.njk
+Use photography when it’s important to show a lifelike representation of something. For example, showing a photo of a document a user has uploaded.
 
+Do not use photography to represent abstract concepts, such as to convey an impression or emotion to the user. Avoid using generic stock photography in your service.
 
-## Alternative text
+## Illustrations
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+Use illustrations to show a simplified image of something complex. For example:
+- to demonstrate a physical interaction, like scanning a passport
+- to help users find something on a document, like a reference number
+
+### Creating illustrations
+
+Use as few visual elements as possible to communicate your message.
+
+Start by using the [GOV.UK colour palette](/styles/colour/#colour-palette). If you need more colours, try to use tints of the colour palette.
+
+Use flat blocks of colour. Avoid using shadows or gradients, except when needed to show a distinctive feature of an object or to communicate layering or depth.
+
+If creating multiple illustrations for a service, use consistent colours, line widths and border radiuses across them all.
+
+Avoid depicting people in illustrations. If you need to show people, think carefully about how you represent the people that use your service — do not only show one kind of person. 
+For example, only depicting people of a particular age, gender or ethnicity can exclude people, reinforce negative stereotypes and cause harm.
+
+Some examples of illustrations from services:
+
+## Icons
+
+Avoid using icons in most cases. People can understand icons to mean different things, so they can be confusing.
+
+Icons can be more useful in case working systems, where users are familiar with the interface and return to it frequently. In this context, they can help users to scan pages more quickly. In most cases it’s still helpful to include a text label alongside any icons.
+
+Do not use a single icon to represent more than one thing. For example, the search icon (magnifying glass) should only be used for search functionality, and not also for screen magnification.
+
+## Avoid images that contain text
+
+Use written content instead of an image that contains text, wherever possible.
+
+Any text inside an image will be difficult to read, especially for users that need to read text using:
+- screen readers
+- copy and paste
+- highlight
+- resize
+- recolour
+
+However, text can sometimes be an inseparable part of an image, especially in photography. Examples of this might be a photo with a signpost in it, or a company's name on the side of a building.
+
+If you choose to use an image that contains text, you should also include written content nearby that conveys the same meaning. This means the information is available to everyone
```

Comment:
There is a difference between 'images of text' and 'images that contain text' from WCAG's point of view. I don't know how to change this text to make that clearer. But I would generally not say that you should always include written content that conveys the same meaning of that text. Text in photos or screenshots can be tangential and might not be important at all. 'Images of text' is about when you use an image instead of text. I think that distinction is quite important.

Reply:
Also, there is a period missing at the end of the last sentence.

Reply:
Ah, I had written this before I got to the section 'Avoid images of text'. That makes some of my previous comment redundant.

Reply:
Thanks @selfthinker. Yes, it's proving a difficult concept to explain. The way I think I understand it from @davidc-gds and [reading w3.org](https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html) is:

- Images that contain text — Any image with some sort of readable text in it. Under which there are:
  - Images of (incidental) text — image with text that is part of a picture that contains significant other visual content
  - Images of (rendered) text — image with text that has been rendered in a non-text form (e.g., an image) in order to achieve a particular visual effect

First, we can defo add more explanation on these terms.

Second, and trickier is finding a set of terms that's clear and consistent. "Images of text" looks like an WCAG definition, so it makes sense to keep that and work from there.

I think another source of challenge is that we're reinterpreting the [Images tutorial on w3.org](https://www.w3.org/WAI/tutorials/images/), which classifies things by the purpose of the text, into giving a practical guide on what to do with specific types of images.